src/dataSources/cloudFirestore/session.js

```diff
@@ -72,6 +72,25 @@ function sessions(dbInstance, logger) {
     return result;
   }
 
+  async function findSessionBySlug(slug) {
+    const docSnap = await sessionsCol
+      .where('sessionSlug', '==', slug.toLowerCase())
+      .where('status', '==', 'ACCEPTED')
+      .get();
+
+    let results = null;
+
+    if (docSnap.size === 1) {
+      dlog('found session by slug %o', slug);
```

Comment:
should be %s since it's just a string and not a object

Reply:
Didn't think it actually mattered for string. Updated